gallery.html

```diff
@@ -2,8 +2,38 @@
 <html>
 <head>
 	<title>Gallery</title>
+	<meta charset="utf-8">
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/footer.css">
 </head>
 <body>
-
+	<footer class="panel-footer">
+    <div class="container">
+      <div class="row">
+        <section id="quick" class="col-lg-4 col-md-4 col-sm-4">
+          <span>Quick Links</span><br><br>
+              <a href="https://www.rsu.ac.in" target="_blank">Raksha Shakti University</a><br>
+              <a href="https://docs.google.com/document/d/1DRbb0VUGORYI9epY3INSEfkWT_SO7EKVQTsieeA3MR4/edit" target="_blank">Code of Conduct</a><br>
+              <a href="https://docs.google.com/document/d/1TDQdYO0Dm0osDrFzyqfe1MjemHMJMdpCQFZVUI4Y-qk/edit" target="_blank">FAQs</a><br>
+        </section>
+        <section id="socia" class="col-lg-4 col-md-4 col-sm-4">
+          <span>Social Links</span><br><br>
+              <a href="https://dsc.community.dev/raksha-shakti-university/" target="_blank">Devloper Student Club</a><br>
+              <a href="https://facebook.com/dsc.rsu.7" target="_blank">Facebook</a><br>
+              <a href="https://www.YouTube.com/dscrsu/" target="_blank">YouTube</a><br>
+              <a href="https://www.linkedin.com/in/dsc-rsu-7b53011b8/" target="_blank">Linkedin</a><br>
+              <a href="https://www.instagram.com/dscrsu/" target="_blank">Instagram</a><br>
+        </section>
+        <section id="Scroll" class="col-lg-4 col-md-4 col-sm-4">
+          <span>Scroll To</span><br><br>
+              <a href="aboutus.html">About Us</a><br>
+              <a href="teams.html">Team</a><br>
+              <a href="projects.html">Projects</a><br>
+              <a href="contactus.html">Contact Us</a><br>
+        </section>
+      </div>
+      <div align="center" style="color: white;"> Copyright&copy; Developers Student Club, Raksha Shakti University</div>
+    </div>
+  </footer>
 </body>
 </html>
```

Comment:
add an extra line over here and every file you've edited

Reply:
between footer and body?

Reply:
> between footer and body?

At the end of the file after `</html>`

Reply:
done